src/main/java/com/spring/boot/service/mapper/ProductMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.spring.boot.service.mapper;
+
+import com.spring.boot.model.Product;
+import com.spring.boot.model.dto.Review;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductMapper {
+    public Product map(Review review) {
```

Comment:
I think you can change the name of the method to something more descriptive like mapProduct, mapAllProducts (below) and the same  - for users.

Reply:
mapToProduct